### ISSUE TITLE
Linkify links in comments when using HTML output

### ISSUE THREAD

**Issue by keegancsmith** [trishume/syntect#136]
Feature request. It would be great to have an optional feature which linkifies part of a comment which contain a link.

**Comment by trishume** [comment]
I agree that could be useful. I'm unlikely to implement it myself though. If you submit a PR I'm happy to review it though!
